packages/ui/src/components/button/button.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import type React from 'react';
```

Comment:
this seems like an odd syntax...is this to import the React type declarations?

Reply:
Correct, since we're only using the type definition of React in these files, we can reduce the overall size of the import by using `import type` which looks to have been introduced back in [Typescript 3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html). This became an issue when I included `"importsNotUsedAsValues": "error"` in the `tsconfig` as it threw errors when we never used React directly.


I think this is the most relevant line in the release notes describing how it's handled.
> `import type` only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime.

Reply:
Very helpful, thank you!